src/app/ach-positive-pay/ach-positive-pay-journey-bundle.module.ts

```diff
@@ -0,0 +1,21 @@
+import { NgModule } from '@angular/core';
+import { AchPositivePayJourneyModule } from 'ach-positive-pay-journey';
+import { CONDITIONS } from '@backbase/foundation-ang/web-sdk';
+import { buildEntitlementsByUser, triplets } from '../services/entitlementsTriplets';
+
+@NgModule({
+  imports: [AchPositivePayJourneyModule.forRoot()],
+  providers: [
+    {
+      provide: CONDITIONS,
```

Comment:
there is no more need to provide this anymore. Now you can add your entitlements here https://github.com/Backbase/golden-sample-app/blob/master/src/app/interceptors/entitlements-interceptor.ts

Reply:
Updated.